app/src/main/java/com/iic/shopingo/ui/trip_flow/views/RequestListAdapter.java

```diff
@@ -15,9 +15,9 @@
   private List<IncomingRequest> requests = new ArrayList<>();
   private Context context;
 
-  public RequestListAdapter(Context context, List<IncomingRequest> requests) {
+  public RequestListAdapter(Context context) {
     this.context = context;
-    this.requests = requests;
+    this.requests = new ArrayList<>(0);
```

Comment:
You can keep the definition initialization, this way if you'll have multiple constructors you won't need to initialize it multiple times.